cooltools/cli/logbin_expected.py

```diff
@@ -0,0 +1,180 @@
+import pandas as pd
+import numpy as np
+from functools import partial
```

Comment:
I added a bunch of stuff to align with other cli-s we have (compute-saddle to be more specific)